app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -118,12 +118,15 @@ class BrowserFragment : Fragment(), BackHandler, UserInteractionHandler {
             requireComponents.toolbar.menuBuilder
         ) { activity?.finish() }
 
+        val findInPageIntegration = FindInPageIntegration(requireComponents.core.sessionManager, findInPageView)
```

Comment:
We have different patterns for integrating our features, I noticed: lateinit class attributes or integration classes that are surrounded by lifecycle.addObserver.

I think we should pick one in the future so it's consistent.

Reply:
Some on them need to get accessed from `onBackPressed`.